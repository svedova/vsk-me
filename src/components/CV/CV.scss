@import "../../style/vars";

$marginSmall: 1rem;
$marginLarge: 2rem;

.cv {
  display: flex;

  .my-history {
    flex: 1 1 auto;
    margin-left: $marginLarge;
  }
}

.cv-section {
  margin-bottom: $marginLarge * 2;

  &:last-child {
    margin-bottom: 0;
  }
}

.sticky-content {
  &:global.is-sticky {
    max-width: 300px;
    padding-right: $marginSmall * 2;
  }
}

.my-details {
  border-radius: 2px;
  background: white;
  flex: 0 0 auto;
  width: 300px;
  padding: $marginSmall;

  .my {
    text-align: center;
    margin-bottom: $marginLarge * 2;

    .my-name {
      font-size: 1.5rem;
    }
  }

  .my-profile-pic {
    margin-bottom: $marginSmall;

    .my-profile-pic-img {
      width: 100%;
    }
  }

  .my-contact {
    .my-contact-row {
      margin-bottom: $marginSmall;

      a {
        color: black;
      }

      :global .fa {
        width: 1.35rem;
        font-size: 1.5rem;
        color: $color-gray;
        margin-right: $marginLarge;
        text-align: center;
        vertical-align: middle;
      }
    }
  }
}

.my-history {
  border-radius: 2px;
  flex: 1 1 auto;
  background-color: white;
  padding: $marginLarge;

  .section {
    margin-top: $marginLarge * 2;

    &:first-child {
      margin-top: 0;
    }
  }

  .section-header {
    $color: $color-blue-alt;
    font-size: 1.5rem;
    padding-bottom: 1rem;
    margin-bottom: 2rem;
    display: flex;
    border-bottom: 1px solid rgba($color, 0.35);
    color: $color;

    :global .fa {
      flex: 0 0 auto;
      color: rgba($color, 0.7);
    }

    .sectionHeaderText {
      flex: 1 1 auto;
      text-align: right;
    }
  }

  .section-text {

  }

  .education {
    margin-bottom: 2rem;

    .education-title {
      font-size: 1.25rem;
    }

    .education-subtitle {
      margin-bottom: 0.25rem;
    }

    .education-date {
      color: $color-gray;
      margin-bottom: 0.5rem;
    }

    &:last-child {
      margin-bottom: 0;
    }
  }

  .experience:not(:last-child) {
    margin-bottom: 2rem;
  }

  .experience-title {
    font-size: 1.25rem;
  }

  .experience-period {
    color: $color-gray;
  }

  .experience-body {
    margin-top: 0.5rem;
  }

  .ps {
    display: flex;
  }

  .ps-left {
    flex: 1 1 48%;
    margin-right: 1%;
  }

  .ps-right {
    flex: 1 1 48%;
    margin-left: 1%;
  }

  .skill-bar {
    border-radius: 1rem;
    border: 1px solid $color-blue-alt;
    overflow: hidden;
    margin-bottom: 0.5rem;
  }

  .skill-bar-bg {
    padding: 0.2rem 1rem;
    background-color: $color-blue-alt;
    font-size: 0.9rem;
    color: white;
  }

  .lang {
    display: flex;
    width: 100%;
    margin-bottom: 0.5rem;
    font-size: 1.2rem;

    &:last-child {
      margin-bottom: 0;
    }

    :global .fa {
      color: $color-blue-alt;
      opacity: 0.5;

      &.fa-star,
      &.fa-star-half-full {
        opacity: 1;
      }
    }
  }

  .lang-name {
    flex: 1 1 auto;
  }

  .lang-stars {
    flex: 0 0 auto;
  }
}