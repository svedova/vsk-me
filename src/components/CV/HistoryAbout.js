import React, { PureComponent } from "react"
import SectionText from "./SectionText"
import SectionHeader from "./SectionHeader"
import classes from "./CV.scss"

export default class HistoryAbout extends PureComponent {
  render() {
    return (
      <div className={classes.section}>
        <SectionHeader fa="fa-user">Summary</SectionHeader>
        <SectionText>
          Being a self-taught passionate programmer, during the recent years I have developed an extreme
          interest into single page apps. Lately, I've been experimenting with Golang as well,
          as its API-based architecture complements the backend for SPAs.
          <br /><br />
          During my professional career as a programmer, I had to deal with complex databases (including setting
          up), writing backend APIs and frontend for several websites.
        </SectionText>
      </div>
    )
  }
}
