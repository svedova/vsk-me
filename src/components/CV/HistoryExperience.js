import React, { PureComponent } from "react"
import PropTypes from "prop-types"
import classes from "./CV.scss"

export default class HistoryExperience extends PureComponent {
  static propTypes = {
    children: PropTypes.node,
    title: PropTypes.string, // The title of the experience
    period: PropTypes.string // The period
  }

  render() {
    const { children, title, period } = this.props

    return (
      <div className={classes.experience}>
        <div className={classes.experienceHeader}>
          <h3 className={classes.experienceTitle}>
            {title}
          </h3>
          <div className={classes.experiencePeriod}>
            <i>
              {period}
            </i>
          </div>
        </div>
        <div className={classes.experienceBody}>
          {children}
        </div>
      </div>
    )
  }
}