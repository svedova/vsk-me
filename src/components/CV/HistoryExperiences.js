import React, { PureComponent } from "react"
import SectionHeader from "./SectionHeader"
import SectionText from "./SectionText"
import Experience from "./HistoryExperience"
import classes from "./CV.scss"

export default class HistoryExperiences extends PureComponent {
  renderTutti() {
    return (
      <Experience title="Tutti.ch" period="2017 July">
        <span>
          I am now working full time for <a href="https://tutti.ch">Tutti.ch</a> as a frontend developer.
          The website is one of the most visited websites in Switzerland with more than&nbsp;
          <b>15 millions</b> monthly page views. In only 5 months I've accomplished the following tasks:
          <ul className={classes.sectionList}>
            <li className={classes.sectionListItem}>
              Increased test coverage from 0% to 44%.
            </li>
            <li className={classes.sectionListItem}>
              Rewrote the whole routing of the frontend (migrate from React Router v2 to v4)
            </li>
            <li className={classes.sectionListItem}>
              Migrated the remaining legacy code to React.
            </li>
            <li className={classes.sectionListItem}>
              Led the project to server side render the whole website (it is right now in the process of being
              tested).
            </li>
          </ul>
        </span>
      </Experience>
    )
  }

  renderCorpax() {
    return (
      <Experience title="Corpax" period="2015 October - 2017 June">
        <span>
          Corpax is a single page team collaboration and communication tool of which I am the idea owner.
          During the project, as a co-founder:
        </span>
        <ul className={classes.sectionList}>
          <li className={classes.sectionListItem}>
            I helped to prepare the business plan which in return
            allowed us to get a <br/><b>CHF 300'000</b> grant from CTI.
          </li>
          <li className={classes.sectionListItem}>
            Pitched the project in St. Gallen's Start Summit
          </li>
          <li className={classes.sectionListItem}>
            Developed the whole application using the following technologies:
            <ul className={classes.sectionList}>
              <li className={classes.sectionListItem}>ES6 (Vue.js, Webpack, Babel, Karma, Jasmine)</li>
              <li className={classes.sectionListItem}>Node.js</li>
              <li className={classes.sectionListItem}>Php</li>
              <li className={classes.sectionListItem}>PostgreSQL</li>
              <li className={classes.sectionListItem}>AWS (EC2, RDS, S3)</li>
            </ul>
          </li>
          <li className={classes.sectionListItem}>
            Also collaborated closely with <b>SUPSI</b> using an agile development methodology
            in a team of 5 people, to build a chat bot.
          </li>
        </ul>
      </Experience>
    )
  }

  renderRegioPress() {
    return (
      <Experience title="RegioPress SA" period="2013 September - 2015 October">
        <span>
          During my employment in this company, I developed the official website of&nbsp;
          <a href="http://laregione.ch" target="_blank">laRegione</a>, which is
          one of the <b>most read</b> newspapers in Ticino, Switzerland.
        </span>
        <span>
          I also created several Wordpress and Drupal websites for an
          agency that we used to work together. Some of the websites are:
        </span>
        <ul className={classes.sectionList}>
          <li className={classes.sectionListItem}>
            <a href="https://www.laregione.ch/" target="_blank">laRegione.ch</a>
          </li>
          <li className={classes.sectionListItem}>
            <a href="http://www.casacosima.ch" target="_blank">casacosima.ch</a>
          </li>
          <li className={classes.sectionListItem}>
            <a href="http://www.commercianti-bellinzona.ch/" target="_blank">commercianti-bellinzona.ch</a>
          </li>
          <li className={classes.sectionListItem}>
            <a href="http://masserialabarca.ch/" target="_blank">masserialabarca.ch</a>
          </li>
          <li className={classes.sectionListItem}>
            <a href="http://www.studiocresto.ch/" target="_blank">studiocresto.ch</a>
          </li>
        </ul>
      </Experience>
    )
  }

  renderMarketKeyfi() {
    return (
      <Experience title="Marketkeyfi.com" period="2010 April - 2012 May">
        <span>
          Using C#, me and another developer, developed this tool which is an online platform for
          shopping from the closest grocery stores in your nearby. You could order from more than
          100 groceries in Istanbul. It was my first startup, unfortunately it is no longer online.
          I was still studying at the same time.
        </span>
      </Experience>
    )
  }

  render() {
    return (
      <div className={classes.section}>
        <SectionHeader fa="fa-briefcase">
          Work experiences
        </SectionHeader>
        <SectionText>
          {this.renderTutti()}
          {this.renderCorpax()}
          {this.renderRegioPress()}
          {this.renderMarketKeyfi()}
        </SectionText>
      </div>
    )
  }
}