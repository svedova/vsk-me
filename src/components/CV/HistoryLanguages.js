import React, { PureComponent } from "react"
import SectionHeader from "./SectionHeader"
import SectionText from "./SectionText"
import classes from "./CV.scss"

export default class HistoryLanguages extends PureComponent {
  static langs = {
    "English": 5,
    "Italian": 5,
    "Turkish": 5,
    "French": 3.5,
    "German": 0.5,
  }

  computeClass(stars, i) {
    const ceil = Math.ceil(stars)

    if (i === ceil && ceil !== stars) {
      return "fa fa-star-half-full"
    }

    if (i <= stars) {
      return "fa fa-star"
    }

    return "fa fa-star-o"
  }

  render() {
    const noOfStars = [1, 2, 3, 4, 5]
    const { langs } = HistoryLanguages

    return (
      <div className={classes.section}>
        <SectionHeader fa="fa-asl-interpreting">
          Languages
        </SectionHeader>
        <SectionText>
          {
            Object.keys(langs).map(lang => (
              <div className={classes.lang} key={lang}>
                <div className={classes.langName}>{lang}</div>
                <div className={classes.langStars}>
                  {noOfStars.map(i => <span className={this.computeClass(langs[lang], i)} key={`s-${i}`}/>)}
                </div>
              </div>
            ))
          }
        </SectionText>
      </div>
    )
  }
}
