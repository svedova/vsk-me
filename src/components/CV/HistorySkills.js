import React, { PureComponent } from "react";
import SectionHeader from "./SectionHeader";
import SectionText from "./SectionText";
import Skill from "./Skill";
import classes from "./CV.scss";

export default class HistorySkills extends PureComponent {
  static skills = [
    ["React, Redux, Vue, Vuex", 95],
    ["Javascript (ES6, Vanilla)", 95],
    ["HTML5/CSS3", 95],
    ["Node.js", 85],
    ["GoLang", 65],
    ["AWS (Lambda, EC2, API GW, Route53, S3)"]
  ];

  static frameworks = [
    ["Python", 55],
    ["Php7", 95],
    ["Bash", 45],
    ["Git", 85],
    ["Docker", 60],
    ["CI/CD (Jenkins, CircleCI)", 80]
  ];

  render() {
    const { skills, frameworks } = HistorySkills;

    return (
      <div className={classes.section}>
        <SectionHeader fa="fa-trophy">
          Programming, Tools & Frameworks
        </SectionHeader>
        <SectionText>
          <div className={classes.ps}>
            <div className={classes.psLeft}>
              {skills.map((s, i) => (
                <Skill name={s[0]} perc={s[1]} key={`s-${i}`} />
              ))}
            </div>
            <div className={classes.psRight}>
              {frameworks.map((s, i) => (
                <Skill name={s[0]} perc={s[1]} key={`f-${i}`} />
              ))}
            </div>
          </div>
        </SectionText>
      </div>
    );
  }
}
