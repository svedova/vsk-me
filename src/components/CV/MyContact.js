import React, { PureComponent } from "react"
import Base64 from "../../helpers/base64"
import classes from "./CV.scss"

export default class MyContact extends PureComponent {
  static FB_LINK = "https://www.facebook.com/savasvedova"
  static IG_LINK = "https://www.instagram.com/savasvedova"

  render() {
    const mail = Base64.decode("c2F2YXNAdnNrLm1l")

    return (
      <div className={classes.myContact}>
        <div className={classes.myContactRow}>
          <a href="https://vsk.me">
            <span className="fa fa-globe"/>
            <span>https://vsk.me</span>
          </a>
        </div>
        <div className={classes.myContactRow}>
          <a href="`mailto:${mail}`">
            <span className="fa fa-envelope"/>
            <span>{mail}</span>
          </a>
        </div>
        <div className={classes.myContactRow}>
          <span className="fa fa-map-pin"/>
          <span>Zurich, Switzerland</span>
        </div>
        <div className={classes.myContactRow}>
          <a href={MyContact.FB_LINK}>
            <span className="fa fa-facebook"/>
            <span>Facebook</span>
          </a>
        </div>
        <div className={classes.myContactRow}>
          <a href={MyContact.IG_LINK}>
            <span className="fa fa-instagram"/>
            <span>Instagram</span>
          </a>
        </div>
      </div>
    )
  }
}
