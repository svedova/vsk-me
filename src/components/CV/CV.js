import React, { Component } from "react"
import { sticky, unsticky } from "sticky-content"
import MyProfile from "./MyProfile"
import MyContact from "./MyContact"
import HistoryAbout from "./HistoryAbout"
import HistoryExperiences from "./HistoryExperiences"
import HistoryEducation from "./HistoryEducation"
import HistorySkills from "./HistorySkills"
import HistoryLanguages from "./HistoryLanguages"
import classes from "./CV.scss"
import CloudFlash from "cloudflash.io-client"

export default class CV extends Component {

  componentDidMount() {
    const root = document.querySelector(`.${classes.stickyContent}`)
    root && sticky(root)
    CloudFlash.persist()
  }

  componentWillUnmount() {
    const root = document.querySelector(`.${classes.stickyContent}`)
    root && unsticky(root)
  }

  render() {
    return (
      <div className={classes.cv}>
        <div className={classes.myDetails}>
          <div className={classes.stickyContent}>
            <MyProfile/>
            <MyContact/>
          </div>
        </div>
        <div className={classes.myHistory}>
          <HistoryAbout/>
          <HistoryExperiences/>
          <HistoryEducation/>
          <HistorySkills/>
          <HistoryLanguages/>
        </div>
      </div>
    )
  }
}
