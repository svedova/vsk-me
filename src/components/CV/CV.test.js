/* global describe, test, expect */
import React from "react"
import CV from "./CV"
import { mount } from "enzyme"

describe("(Components) CV", () => {
  test("should match the snapshot", () => {
    const comp = mount(<CV/>)
    expect(comp).toMatchSnapshot()
  })
})