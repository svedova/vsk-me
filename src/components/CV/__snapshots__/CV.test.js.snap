// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Components) CV should match the snapshot 1`] = `
<div
  className={undefined}
>
  <div
    className={undefined}
  >
    <div
      className={undefined}
    >
      <div
        className={undefined}
      >
        <div
          className={undefined}
        >
          <img
            className={undefined}
            src="me.png"
          />
        </div>
        <div
          className={undefined}
        >
          <h1
            className={undefined}
          >
            SAVAS VEDOVA
          </h1>
          <i
            className={undefined}
          >
            Full stack developer
          </i>
        </div>
      </div>
      <div
        className={undefined}
      >
        <div
          className={undefined}
        >
          <a
            href="https://vsk.me"
          >
            <span
              className="fa fa-globe"
            />
            <span>
              https://vsk.me
            </span>
          </a>
        </div>
        <div
          className={undefined}
        >
          <a
            href="\`mailto:\${mail}\`"
          >
            <span
              className="fa fa-envelope"
            />
            <span>
              savas@vsk.me
            </span>
          </a>
        </div>
        <div
          className={undefined}
        >
          <span
            className="fa fa-map-pin"
          />
          <span>
            Zurich, Switzerland
          </span>
        </div>
        <div
          className={undefined}
        >
          <a
            href="https://www.facebook.com/savasvedova"
          >
            <span
              className="fa fa-facebook"
            />
            <span>
              Facebook
            </span>
          </a>
        </div>
        <div
          className={undefined}
        >
          <a
            href="https://www.instagram.com/savasvedova"
          >
            <span
              className="fa fa-instagram"
            />
            <span>
              Instagram
            </span>
          </a>
        </div>
      </div>
    </div>
  </div>
  <div
    className={undefined}
  >
    <div
      className={undefined}
    >
      <h2
        className={undefined}
      >
        <span
          className="fa fa-user"
        />
        <span
          className={undefined}
        >
          Summary
        </span>
      </h2>
      <div
        className={undefined}
      >
        Being a self-taught passionate programmer, during the recent years I have developed an extreme interest into single page apps. Lately, I've been experimenting with Golang as well, as its API-based architecture complements the backend for SPAs.
        <br />
        <br />
        During my professional career as a programmer, I had to deal with complex databases (including setting up), writing backend APIs and frontend for several websites.
      </div>
    </div>
    <div
      className={undefined}
    >
      <h2
        className={undefined}
      >
        <span
          className="fa fa-briefcase"
        />
        <span
          className={undefined}
        >
          Work experiences
        </span>
      </h2>
      <div
        className={undefined}
      >
        <div
          className={undefined}
        >
          <div
            className={undefined}
          >
            <h3
              className={undefined}
            >
              Tutti.ch
            </h3>
            <div
              className={undefined}
            >
              <i>
                2017 July
              </i>
            </div>
          </div>
          <div
            className={undefined}
          >
            <span>
              I am now working full time for 
              <a
                href="https://tutti.ch"
              >
                Tutti.ch
              </a>
               as a frontend developer. The website is one of the most visited websites in Switzerland with more thanÂ 
              <b>
                15 millions
              </b>
               monthly page views. In only 5 months I've accomplished the following tasks:
              <ul
                className={undefined}
              >
                <li
                  className={undefined}
                >
                  Rewrote the whole routing of the frontend (migrate from React Router v2 to v4)
                </li>
                <li
                  className={undefined}
                >
                  Migrated the remaining legacy code to React.
                </li>
                <li
                  className={undefined}
                >
                  Led the project to server side render the whole website (it is right now in the process of being tested).
                </li>
              </ul>
            </span>
          </div>
        </div>
        <div
          className={undefined}
        >
          <div
            className={undefined}
          >
            <h3
              className={undefined}
            >
              Corpax
            </h3>
            <div
              className={undefined}
            >
              <i>
                2015 October - 2017 June
              </i>
            </div>
          </div>
          <div
            className={undefined}
          >
            <span>
              Corpax is a single page team collaboration and communication tool of which I am the idea owner. During the project, as a co-founder:
            </span>
            <ul
              className={undefined}
            >
              <li
                className={undefined}
              >
                I helped to prepare the business plan which in return allowed us to get a 
                <br />
                <b>
                  CHF 300'000
                </b>
                 grant from CTI.
              </li>
              <li
                className={undefined}
              >
                Pitched the project in St. Gallen's Start Summit
              </li>
              <li
                className={undefined}
              >
                Developed the whole application using the following technologies:
                <ul
                  className={undefined}
                >
                  <li
                    className={undefined}
                  >
                    ES6 (Vue.js, Webpack, Babel, Karma, Jasmine)
                  </li>
                  <li
                    className={undefined}
                  >
                    Node.js
                  </li>
                  <li
                    className={undefined}
                  >
                    Php
                  </li>
                  <li
                    className={undefined}
                  >
                    PostgreSQL
                  </li>
                  <li
                    className={undefined}
                  >
                    AWS (EC2, RDS, S3)
                  </li>
                </ul>
              </li>
              <li
                className={undefined}
              >
                Also collaborated closely with 
                <b>
                  SUPSI
                </b>
                 using an agile development methodology in a team of 5 people, to build a chat bot.
              </li>
            </ul>
          </div>
        </div>
        <div
          className={undefined}
        >
          <div
            className={undefined}
          >
            <h3
              className={undefined}
            >
              RegioPress SA
            </h3>
            <div
              className={undefined}
            >
              <i>
                2013 September - 2015 October
              </i>
            </div>
          </div>
          <div
            className={undefined}
          >
            <span>
              During my employment in this company, I developed the official website of
              <a
                href="http://laregione.ch"
                target="_blank"
              >
                laRegione
              </a>
              , which is one of the 
              <b>
                most read
              </b>
               newspapers in Ticino, Switzerland.
            </span>
            <span>
              I also created several Wordpress and Drupal websites for an agency that we used to work together. Some of the websites are:
            </span>
            <ul
              className={undefined}
            >
              <li
                className={undefined}
              >
                <a
                  href="http://www.casacosima.ch"
                  target="_blank"
                >
                  casacosima.ch
                </a>
              </li>
              <li
                className={undefined}
              >
                <a
                  href="http://www.commercianti-bellinzona.ch/"
                  target="_blank"
                >
                  commercianti-bellinzona.ch
                </a>
              </li>
              <li
                className={undefined}
              >
                <a
                  href="http://masserialabarca.ch/"
                  target="_blank"
                >
                  masserialabarca.ch
                </a>
              </li>
              <li
                className={undefined}
              >
                <a
                  href="http://www.studiocresto.ch/"
                  target="_blank"
                >
                  studiocresto.ch
                </a>
              </li>
            </ul>
          </div>
        </div>
        <div
          className={undefined}
        >
          <div
            className={undefined}
          >
            <h3
              className={undefined}
            >
              Marketkeyfi.com
            </h3>
            <div
              className={undefined}
            >
              <i>
                2010 April - 2012 May
              </i>
            </div>
          </div>
          <div
            className={undefined}
          >
            <span>
              Using C#, me and another developer, developed this tool which is an online platform for shopping from the closest grocery stores in your nearby. You could order from more than 100 groceries in Istanbul. It was my first startup, unfortunately it is no longer online. I was still studying at the same time.
            </span>
          </div>
        </div>
      </div>
    </div>
    <div
      className={undefined}
    >
      <h2
        className={undefined}
      >
        <span
          className="fa fa-graduation-cap"
        />
        <span
          className={undefined}
        >
          Education
        </span>
      </h2>
      <div
        className={undefined}
      >
        <div
          className={undefined}
        >
          <h3
            className={undefined}
          >
            SUPSI
          </h3>
          <h4
            className={undefined}
          >
            <i>
              Computer Science
            </i>
          </h4>
          <div
            className={undefined}
          >
            2013 - 2014
          </div>
          <div
            className={undefined}
          >
            Studied computer science to get a bachelor degree in my field. However, I dropped out once RegioPress decided to finance Corpax.
          </div>
        </div>
        <div
          className={undefined}
        >
          <h3
            className={undefined}
          >
            Bogazici University
          </h3>
          <h4
            className={undefined}
          >
            <i>
              Political Science and International Relations
            </i>
          </h4>
          <div
            className={undefined}
          >
            2007 - 2012
          </div>
          <div
            className={undefined}
          >
            Bogazici is one of the best universities in Turkey. My department required to be in the first 700 among 1.5m students in a nation-wide exam. I was the 559th.
          </div>
        </div>
      </div>
    </div>
    <div
      className={undefined}
    >
      <h2
        className={undefined}
      >
        <span
          className="fa fa-trophy"
        />
        <span
          className={undefined}
        >
          Programming, Tools & Frameworks
        </span>
      </h2>
      <div
        className={undefined}
      >
        <div
          className={undefined}
        >
          <div
            className={undefined}
          >
            <div
              className={undefined}
            >
              <div
                className={undefined}
                style={
                  Object {
                    "width": "95%",
                  }
                }
              >
                Javascript (ES6, Vanilla)
              </div>
            </div>
            <div
              className={undefined}
            >
              <div
                className={undefined}
                style={
                  Object {
                    "width": "95%",
                  }
                }
              >
                Php7
              </div>
            </div>
            <div
              className={undefined}
            >
              <div
                className={undefined}
                style={
                  Object {
                    "width": "95%",
                  }
                }
              >
                HTML5/CSS3
              </div>
            </div>
            <div
              className={undefined}
            >
              <div
                className={undefined}
                style={
                  Object {
                    "width": "90%",
                  }
                }
              >
                SQL (PostgreSQL, MySQL)
              </div>
            </div>
            <div
              className={undefined}
            >
              <div
                className={undefined}
                style={
                  Object {
                    "width": "45%",
                  }
                }
              >
                Bash
              </div>
            </div>
            <div
              className={undefined}
            >
              <div
                className={undefined}
                style={
                  Object {
                    "width": "70%",
                  }
                }
              >
                Node.js
              </div>
            </div>
            <div
              className={undefined}
            >
              <div
                className={undefined}
                style={
                  Object {
                    "width": "55%",
                  }
                }
              >
                Python
              </div>
            </div>
            <div
              className={undefined}
            >
              <div
                className={undefined}
                style={
                  Object {
                    "width": "30%",
                  }
                }
              >
                C#
              </div>
            </div>
            <div
              className={undefined}
            >
              <div
                className={undefined}
                style={
                  Object {
                    "width": "45%",
                  }
                }
              >
                Java
              </div>
            </div>
          </div>
          <div
            className={undefined}
          >
            <div
              className={undefined}
            >
              <div
                className={undefined}
                style={
                  Object {
                    "width": "95%",
                  }
                }
              >
                React, Redux
              </div>
            </div>
            <div
              className={undefined}
            >
              <div
                className={undefined}
                style={
                  Object {
                    "width": "95%",
                  }
                }
              >
                Vue, Vuex
              </div>
            </div>
            <div
              className={undefined}
            >
              <div
                className={undefined}
                style={
                  Object {
                    "width": "80%",
                  }
                }
              >
                Drupal
              </div>
            </div>
            <div
              className={undefined}
            >
              <div
                className={undefined}
                style={
                  Object {
                    "width": "80%",
                  }
                }
              >
                Wordpress
              </div>
            </div>
            <div
              className={undefined}
            >
              <div
                className={undefined}
                style={
                  Object {
                    "width": "70%",
                  }
                }
              >
                Bootstrap
              </div>
            </div>
            <div
              className={undefined}
            >
              <div
                className={undefined}
                style={
                  Object {
                    "width": "70%",
                  }
                }
              >
                Materialize
              </div>
            </div>
            <div
              className={undefined}
            >
              <div
                className={undefined}
                style={
                  Object {
                    "width": "80%",
                  }
                }
              >
                jQuery
              </div>
            </div>
            <div
              className={undefined}
            >
              <div
                className={undefined}
                style={
                  Object {
                    "width": "90%",
                  }
                }
              >
                npm/yarn
              </div>
            </div>
            <div
              className={undefined}
            >
              <div
                className={undefined}
                style={
                  Object {
                    "width": "85%",
                  }
                }
              >
                git
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className={undefined}
    >
      <h2
        className={undefined}
      >
        <span
          className="fa fa-asl-interpreting"
        />
        <span
          className={undefined}
        >
          Languages
        </span>
      </h2>
      <div
        className={undefined}
      >
        <div
          className={undefined}
          key="English"
        >
          <div
            className={undefined}
          >
            English
          </div>
          <div
            className={undefined}
          >
            <span
              className="fa fa-star"
              key="s-1"
            />
            <span
              className="fa fa-star"
              key="s-2"
            />
            <span
              className="fa fa-star"
              key="s-3"
            />
            <span
              className="fa fa-star"
              key="s-4"
            />
            <span
              className="fa fa-star"
              key="s-5"
            />
          </div>
        </div>
        <div
          className={undefined}
          key="Italian"
        >
          <div
            className={undefined}
          >
            Italian
          </div>
          <div
            className={undefined}
          >
            <span
              className="fa fa-star"
              key="s-1"
            />
            <span
              className="fa fa-star"
              key="s-2"
            />
            <span
              className="fa fa-star"
              key="s-3"
            />
            <span
              className="fa fa-star"
              key="s-4"
            />
            <span
              className="fa fa-star"
              key="s-5"
            />
          </div>
        </div>
        <div
          className={undefined}
          key="Turkish"
        >
          <div
            className={undefined}
          >
            Turkish
          </div>
          <div
            className={undefined}
          >
            <span
              className="fa fa-star"
              key="s-1"
            />
            <span
              className="fa fa-star"
              key="s-2"
            />
            <span
              className="fa fa-star"
              key="s-3"
            />
            <span
              className="fa fa-star"
              key="s-4"
            />
            <span
              className="fa fa-star"
              key="s-5"
            />
          </div>
        </div>
        <div
          className={undefined}
          key="French"
        >
          <div
            className={undefined}
          >
            French
          </div>
          <div
            className={undefined}
          >
            <span
              className="fa fa-star"
              key="s-1"
            />
            <span
              className="fa fa-star"
              key="s-2"
            />
            <span
              className="fa fa-star"
              key="s-3"
            />
            <span
              className="fa fa-star-half-full"
              key="s-4"
            />
            <span
              className="fa fa-star-o"
              key="s-5"
            />
          </div>
        </div>
        <div
          className={undefined}
          key="German"
        >
          <div
            className={undefined}
          >
            German
          </div>
          <div
            className={undefined}
          >
            <span
              className="fa fa-star-half-full"
              key="s-1"
            />
            <span
              className="fa fa-star-o"
              key="s-2"
            />
            <span
              className="fa fa-star-o"
              key="s-3"
            />
            <span
              className="fa fa-star-o"
              key="s-4"
            />
            <span
              className="fa fa-star-o"
              key="s-5"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
